src/modules/fw_pos_control/FixedwingPositionControl.cpp

```diff
@@ -504,6 +505,14 @@ FixedwingPositionControl::landing_status_publish()
 	_pos_ctrl_landing_status_pub.publish(pos_ctrl_landing_status);
 }
 
+float FixedwingPositionControl::getCorrectedNpfgRollSetpoint()
```

Comment:
could we move that all into NPFG::updateRollSetpoint()? Then also the slew rate limiting could be done correctly.

Reply:
I would prefer it outside, as the npfg controller should not do the error handling, rather the upper layers. On the other hand, does the slew rate need to be in the npfg? I could move that outside as well to have it at the same place.

Reply:
I don't have a strong preference, but would rather have done both inside personally. Leave it to you.